### ISSUE TITLE
Sultan RS V8 buff

### ISSUE THREAD

**Issue by CreepyEK** [ChainedDev/CityLife#869]
**What is the reason for this balance request?**
The car is simply not competitive with any of the new S class imports. slower, and handling just isn't up to par.

**What solution would you like?**
A re-evaluation of this and perhaps some of the older imports that people have spent a lot of money on, and bring them up to speed with the newer ones like the 190E, widebody Jester ect.

**Additional context**
Things like the RX8 are also extremely underpowered in comparison to the cars mentioned above.

**Comment by Forkedup** [comment]
I can understand your frustration with the performance of your car compared to the newer S class imports. It sounds like you would like to see a re-evaluation of the older imports to improve their performance and bring them up to speed with the newer models. We have slowly been tuning down performance of the newer cars as of last week.

**Comment by CreepyEK** [comment]
Yeah, a re-evaluation of the older ones would definitely be appreciated, as myself and others with other older cars feel like ours just seem a decent amount worse.

**Comment by CreepyEK** [comment]
After testing the changes, the RX8 feels perfect, competitive and great for racing, however the sultan lacks a bit still with handling, it slides out and feels like the rear of the car loses grip easy and it's hard to get it around corners.
